test/real_content/real_content_lib.bash

```diff
@@ -57,7 +57,7 @@ install_bundles() {
 		BUNDLE_LIST=$(echo "$output" | tr '\n' ' ')
 	fi
 
-	echo "Install bundles: $BUNDLE_LIST"
+	print "Install bundles: $BUNDLE_LIST"
```

Comment:
@castulo @otaviobp I ran passing/failing tests and I do not see output from the 'print' command. Does bats actually show the output of 'print' or am I missing something here?

Reply:
bats will show that output on passing test when using the _-t_ option (bats -t your_test.bats), or it will show it in failing tests without using -t

Reply:
ahhh I see